migrations/Version20210118135848.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20210118135848 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return '';
```

Comment:
Why there is a method which returns an empty string?

Reply:
Resolve